ruby/anagram.rb

```diff
@@ -0,0 +1,16 @@
+class Anagram
+  attr_reader :root_word
+
+  def initialize(root_word)
+    @root_word = root_word
+  end
+
+  def match(candidates)
+    anagrams = []
+    for c in candidates do
+      anagrams.push(c) if (c.downcase != self.root_word.downcase) && (c.downcase.split(//).sort == self.root_word.downcase.split(//).sort)
+    end
+    anagrams
```

Comment:
Is there a way to utilize ruby's implicit returns to avoid explicity returning the anagrams variable?